addons/options/src/preview/index.js

```diff
@@ -41,3 +41,32 @@ export function setOptions(newOptions) {
 
   channel.emit(EVENT_ID, { options });
 }
+
+export const withOptions = makeDecorator({
+  name: 'withOptions',
+  parameterName: 'options',
+  skipIfNoParametersOrOptions: false,
+  allowDeprecatedUsage: true,
```

Comment:
@nm123github -- was there a particular reason why you decided to allow deprecated usage for this decorator?

Reply:
Just noticed bunch of other add-ons that had set it to true, so presumed this was the pattern to follow and maybe there was a plan to set them to false at a later date.

Reply:
Gotcha. That option is to allow `add('story', withOptions(() => <Story />))` style usage (which is the old way addons did it pre-4.0). Given this API never existed for `withOptions` I don't think we need it.